x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -435,7 +474,11 @@ export function AlertsTableTGrid(props: AlertsTableTGridProps) {
           <LazyAlertsFlyout
             alert={flyoutAlert}
             observabilityRuleTypeRegistry={observabilityRuleTypeRegistry}
-            onClose={handleFlyoutClose}
+            onClose={onCollapseFn}
+            onSelectedAlertIndexChange={setSelectedAlertIndex}
```

Comment:
@ersinerdal it's passed here

Reply:
i mean `selectedAlertId` not `onSelectedAlertIndexChange` :)

Reply:
Oh sorry I misunderstood. See my comment above.